.github/workflows/build_and_upload_docker_image_dev.yml

```diff
@@ -2,7 +2,7 @@ name: Build and Upload Docker Image of Current Dev Branch to GHCR
 
 on:
   schedule:
-    - cron: "0 16 * * 1"  # Weekly at noon EST on Monday
+    - cron: "0 13 * * *" # Daily at 9 EST
```

Comment:
Os Is this temporary or you want to push more often?

Reply:
More often when it replaces rather than piles on is better

However, doing a fresh build on merge to main would actually be best

Will have separate PR for that